citest/gcp_testing/google_session.py

```diff
@@ -69,13 +77,17 @@ def __init__(
         approval_dict[input['name']] = input['value']
     approval_dict['submit_access'] = 'true'
     self.__session.post(approval_form['action'], data=approval_dict)
```

Comment:
I'm not diggin' the logout_url here or the flow in logout(). At least not now. It is hooking things together without being clear with what this means or is for or why the caller doesnt. On its own, this is authenticating a session. It isnt intrinsically on behalf of another party. So to logout for another party is just weird, and adding a registry of people to notify is overkill without more justification.